### PR TITLE
Comments for Detail View

### PR DESCRIPTION
**Pull request by mcolombo87**
## Summary

comments component and add that in detail view. Comments was limited to 2 elements, if click in View All you can see all comments available.

IMAGE ALT TEXT HERE

## Screenshots
**Nexus 5X de 5.2" Android 10 (API 29)**
image

**Pixel 3 XL 6.3 PIE (9)**
image

**Pixel 2 de 5" Nogaout (7.1)**
image

**Pantalla 5.4" Lollipop (5.1)**
image

## Trello Card
https://trello.com/c/47hHnLNR/16-tabla-de-comentarios

### CODE REVIEW

File: src/app/screens/BookDetail/components/Comment/styles.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    flex: 3,
+    flexDirection: 'row',
+    width: 335,
+    borderRadius: 5,
+    backgroundColor: colors.white,
+    padding: 15
+  },
+  avatar: {
+    height: 45,
+    width: 45,
```
Comment (guidoprinc) [inline src/app/screens/BookDetail/components/Comment/styles.js:15]: extract value as a constant

File: src/app/screens/BookDetail/index.tsx [hunk 2]
```diff
@@ -10,11 +12,46 @@ interface Props {
   };
 }
 
+interface dataComments {
+  item: {
+    id: number;
+    name: string;
+    lastname: string;
+    comment: string;
+    avatarImage: string | null;
```
Comment (guidoprinc) [inline src/app/screens/BookDetail/index.tsx:21]: you set this props to any in Comment. Change Comment prop type

File: src/app/screens/BookDetail/index.tsx [hunk 3]
```diff
@@ -10,11 +12,46 @@ interface Props {
...
+  const keyExtractor = (item: { id: number }) => `${item.id}`;
+  const renderItem = ({ item: { name, lastname, comment, avatarImage } }: dataComments) => (
+    <Comment name={name} lastName={lastname} comment={comment} avatarImage={avatarImage} />
+  );
+  const comments = fullComments ? Comments : Comments.slice(0, 2);
+  const toggleComments = () => {
+    setFullComments(!fullComments);
+  };
+  const selectTextForComments = fullComments ? 'Hide Comments' : 'View All';
+  const listFotter = () => {
```
Comment (guidoprinc) [inline src/app/screens/BookDetail/index.tsx:37]: footer

File: src/app/screens/BookDetail/components/Comment/styles.js (hunk 1 above)
Comment (mcolombo87) [reply src/app/screens/BookDetail/components/Comment/styles.js]: Done!

File: src/app/screens/BookDetail/index.tsx (hunk 2 above)
Comment (mcolombo87) [reply src/app/screens/BookDetail/index.tsx:21]: The error was in the "any" declaration, i changed that.

File: src/app/screens/BookDetail/index.tsx (hunk 3 above)
Comment (mcolombo87) [reply src/app/screens/BookDetail/index.tsx]: Fixed

**Pull request merged**